make_natnum(marker_position(x) as i64) >expected i32, found i64

Hi,

Windows 10, latest Rust version installed I run into the arch problem but don't know really why?!

[C:\workspace\rust\Projekte\remacs-master\rust_src]cargo build
   Compiling lazy_static v0.2.2
   Compiling libc v0.2.17
   Compiling remacs v0.1.0 (file:///C:/workspace/rust/Projekte/remacs-master/rust_src)
error[E0308]: mismatched types
   --> src\lisp.rs:316:21
    |
316 |         make_natnum(marker_position(x) as i64)
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^ expected i32, found i64

error: aborting due to previous error

error: Could not compile `remacs`.

To learn more, run the command again with --verbose.

--- a/rust_src/src/lisp.rs
+++ b/rust_src/src/lisp.rs
@@ -313,7 +313,7 @@ fn test_numberp() {
 /// value rather than assigning to a variable.
 pub fn check_number_coerce_marker(x: LispObject) -> LispObject {
     if MARKERP(x) {
-        make_natnum(marker_position(x) as i64)
+        make_natnum(marker_position(x) as EmacsInt)
     } else {
         unsafe {
             CHECK_TYPE(NUMBERP(x), Qnumber_or_marker_p, x);

